endpoints/openrtb2/video_auction.go

```diff
@@ -514,18 +562,37 @@ func (deps *endpointDeps) validateVideoRequest(req *openrtb_ext.BidRequestVideo)
 		err := errors.New("request missing required field: PodConfig.Pods")
 		errL = append(errL, err)
 	}
+	podErrors := make([]PodError, 0, 0)
+	podIdsSet := make(map[int]bool)
 	for ind, pod := range req.PodConfig.Pods {
+		podErr := PodError{}
+
+		if podIdsSet[pod.PodId] == true {
+			err := fmt.Sprintf("request duplicated required field: PodConfig.Pods.PodId, Pod id: %d", pod.PodId)
+			podErr.PodErrors = append(podErr.PodErrors, err)
+		} else {
+			podIdsSet[pod.PodId] = true
+		}
 		if pod.PodId <= 0 {
-			err := fmt.Errorf("request missing required field: PodConfig.Pods.PodId, Pod index: %d", ind)
-			errL = append(errL, err)
+			err := fmt.Sprintf("request missing required field: PodConfig.Pods.PodId, Pod index: %d", ind)
+			podErr.PodErrors = append(podErr.PodErrors, err)
```

Comment:
We need to make sure we document that PodId must be positive (or change type to unsigned int and say document not to use zero) if negative/zero values indicate an error.